README.md

```diff
@@ -0,0 +1,41 @@
+# cyrus-docker
+
+This image contains a cyrus imap service.
+
+There's three users belonging to a virtual test domain :

```

Comment:
Three users belong to a virtual test domain:

Is it just any virtual domain, or is there one specific domain? In the latter case, replace "a" with "the".